examples/lib/tour/01-getting-started/README.md

```diff
@@ -3,11 +3,265 @@
+This is typically done by a `defer` statement placed after a
+successful object retrieval.
+
+```go
+	defer repo.Close()
+```
+
+Now we look for the versions of the component
+available in this repository.
+
+```go
+	versions, err := c.ListVersions()
+	if err != nil {
+		return errors.Wrapf(err, "cannot query version names")
+	}
+```
+
+OCM version names must follow the semver rules.
+Therefore, we can simply order the versions and print them.
+
+```go
+	err = semverutils.SortVersions(versions)
+	if err != nil {
+		return errors.Wrapf(err, "cannot sort versions")
+	}
+	fmt.Printf("versions for component ocm.software/ocmcli: %s\n", strings.Join(versions, ", "))
+```
+
+Now, we have a look at the latest version, it is
+the last one in the list.
+
+```go
+	cv, err := c.LookupVersion(versions[len(versions)-1])
+	if err != nil {
+		return errors.Wrapf(err, "cannot get latest version")
+	}
+	defer cv.Close()
+```
+
+<a id="describe-version"></a>
+
+The component version object provides access
+to the component descriptor
+
+```go
+	cd := cv.GetDescriptor()
+	fmt.Printf("resources of the latest version:\n")
+	fmt.Printf("  version:  %s\n", cv.GetVersion())
+	fmt.Printf("  provider: %s\n", cd.Provider.Name)
+```
+
+and the resources described by the component version.
+
+```go
+	for i, r := range cv.GetResources() {
+		fmt.Printf("  %2d: name:           %s\n", i+1, r.Meta().GetName())
+		fmt.Printf("      extra identity: %s\n", r.Meta().GetExtraIdentity())
+		fmt.Printf("      resource type:  %s\n", r.Meta().GetType())
+		acc, err := r.Access()
+		if err != nil {
+			fmt.Printf("      access:         error: %s\n", err)
+		} else {
+			fmt.Printf("      access:         %s\n", acc.Describe(ctx))
+		}
+	}
+```
+
+This results in the following output (the shown version might
+differ, because the code always describes the latest version):
+
+```
+resources of the latest version:
+  version:  0.6.0
+  provider: ocm.software
+   1: name:           ocmcli
+      extra identity: "architecture"="amd64","os"="linux"
+      resource type:  executable
+      access:         Local blob sha256:6672528b57fd77cefa4c5a3395431b6a5aa14dc3ddad3ffe52343a7a518c2cd3[]
+   2: name:           ocmcli
+      extra identity: "architecture"="arm64","os"="linux"
+      resource type:  executable
+      access:         Local blob sha256:9088cb8bbef1593b905d6bd3af6652165ff82cebd0d86540a7be9637324d036b[]
+   3: name:           ocmcli-image
+      extra identity: 
+      resource type:  ociImage
+      access:         OCI artifact ghcr.io/open-component-model/ocm/ocm.software/ocmcli/ocmcli-image:0.6.0
+```
+
+Resources have some metadata, like the resource identity and a resource type.
+And they describe how the content of the resource (as blob) can be accessed.
+This is done by an *access specification*, again a serializable descriptor,
+like the respository specification.
+
+The component version contains the executables for the OCM CLI
```

Comment:
sollte man hier nicht eventuell das "component version" als resource in Hochticks stellen oder noch so etwas wie "die Resource" oder "der Typ" oder was component version hier auch immer ist?

Reply:
There are more errors, here, fo example, the identity is not stored as label. I've reformulated the paragraph.